Cargo.toml

```diff
@@ -123,6 +124,17 @@ chacha20poly1305 = { version = "0.7.1", optional = true }
   optional = true
   features = [ "derive" ]
 
+  [dependencies.k256]
```

Comment:
Most other dependencies are in one-line form: `crate = { version = "1.1",  optional = true }`. Let's keep that style